packages/opencensus-instrumentation-http/src/http.ts

```diff
@@ -228,7 +228,12 @@ export class HttpPlugin extends BasePlugin {
               rootSpan.addAttribute(
                   HttpPlugin.ATTRIBUTE_HTTP_ROUTE, requestUrl.path || '');
               tags.set(
-                  stats.HTTP_SERVER_ROUTE, {value: requestUrl.path || ''},
+                  stats.HTTP_SERVER_ROUTE, {
+                    value: ((requestUrl.path && requestUrl.path.length > 255) ?
```

Comment:
Can this be simplified to just `(requestUrl.path || '').substring(0, 255)`?

Calling substring on a shorter string just returns the string so for strings shorter than 255 that is fine.

Should 255 be a constant as well?

Reply:
Sure, done. I was thinking not to use `substring` unless it is really required by checking the length.

Reply:
Makes sense - I'd say either is fine. I'm not sure whether the `substring` has a performance/memory penalty, but if it did that seems like a reason to only do it if needed.

Reply:
Not sure, already made the change as per your suggestion.